### PR TITLE
Add mitodl/xblock-utils as dependency

### PR DESCRIPTION
**Pull request by noisecapella**
Blocked by #188

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [pytest]
 addopts = --cov . --pep8 --pylint --ds=edx_sga.test_settings
-norecursedirs = node_modules .git .tox edx_sga.static edx_sga.locale edx_sga.templates {arch} *.egg
+norecursedirs = __pycache__ build node_modules .git .tox edx_sga.static edx_sga.locale edx_sga.templates {arch} *.egg
```
Comment (amir-qayyum-khan) [inline pytest.ini]: SGA dont have 
- node_modules

can you add `edx_sga.egg-info`, `.cache`

Comment (amir-qayyum-khan) [reply pytest.ini]: what is `__pycache__ build`?

File: setup.py [hunk 2]
```diff
@@ -28,6 +28,10 @@ def package_data(pkg, root_list):
     include_package_data=True,
     install_requires=[
         'XBlock',
+        'mitodl_xblock_utils==1.0.5',
```
Comment (amir-qayyum-khan) [inline setup.py]: why `mitodl_xblock_utils`, it is working fine without it. is this part of this PR scope?

File: pytest.ini (hunk 1 above)
Comment (noisecapella) [reply pytest.ini]: I had these directories locally but I'm not seeing them anymore, I'll remove them from the list

File: setup.py (hunk 2 above)
Comment (noisecapella) [reply setup.py]: Yes, the PR's scope is to add xblock-utils as a dependency, specifically our fork of it. It is a requirement for #185

**Comment by noisecapella** [comment]
Comments addressed

**Pull request merged**